app/models/work_package/pdf_export/work_package_detail.rb

```diff
@@ -95,54 +91,62 @@ def attributes_table_column_widths
     widths.map { |w| w * ratio }
   end
 
-  def build_columns_table_rows(work_package)
-    list = column_objects.reject { |column| column.name == :subject }
+  def attribute_table_rows(work_package)
+    list = attribute_data_list(work_package)
     0.step(list.length - 1, 2).map do |i|
-      build_columns_table_cells(list[i], work_package) +
-        build_columns_table_cells(list[i + 1], work_package)
+      build_columns_table_cells(list[i]) +
+        build_columns_table_cells(list[i + 1])
     end
   end
 
-  def build_attributes_table_rows(work_package)
-    # get work package attribute table rows data [[label, value, label, value]]
-    attrs = %i[
+  def attribute_data_list(work_package)
+    list = if respond_to?(:column_objects)
+             attributes_list_by_columns(work_package)
+           else
+             attributes_list_by_wp(work_package)
+           end
```

Comment:
It seems these don't need `work_package` as an argument at all. Perhaps

```suggestion
    list = if respond_to?(:column_objects)
             attributes_list_by_columns
           else
             attributes_list_by_wp
           end
```